### PR TITLE
Feature/new button component

### PR DESCRIPTION
**Pull request by nswaldman**
The lightning rod for the practical work in https://github.com/demozoo/demozoo/pull/580 -it brings buttons!

This PR will:

- Move and consolidate all sorts of buttons -be it anchor links, button elements or input buttons, to a single, reusable <del>component</del> snippet. Its look & feel needs to be nailed down still.
- Consolidate block and partial templates into something a little more consistent
- Introduce an entity partial for rendering people, groups, events
- Consolidate listings of the previous in the list organism
- Improve consistency of maintenance and general releaser/production overview pages

### CODE REVIEW

**Comment by nswaldman** [comment]
- [ ] Add vanilla button styles
- [ ] Document how people should engage the button component
- [ ] Write tests
- [ ] Apply on production pages based on #580

**Comment by nswaldman** [comment]
The WIP already contains a few things worth discussing:

1. This update would open the door to aBEM conventions for templates and subsequent styling. From experience these are positives for a project like demozoo, but I recognise it might need an elevator pitch.
2. This is a first-time conversion from a Nunjucks macro to a DJT include and I'm not happy with how cumbersome passing data to an include seems to be in comparison with template languages inspired by DJT. Insights welcome.

File: src/style/_legacy/components/productions/credits_panel.scss [hunk 1]
```diff
@@ -1,26 +1,3 @@
-.credits_panel {
```
Comment (nswaldman) [inline src/style/_legacy/components/productions/credits_panel.scss:1]: Removing this implies moving the layout of these types should include having icons like results.

**Comment by gasman** [comment]
I think I have to put this PR out of its misery, sorry - even if it does get finished, it's not going to be reviewable in this form. Please try to stick to fixing one thing at a time...

**Pull request closed without merging**